packages/regl-worldview/src/commands/Command.js

```diff
@@ -127,35 +138,71 @@ export default class Command<T> extends React.Component<Props<T>> {
   }
 }
 
-function getHitmapProps(getHitmapId, children) {
+function defaultGetHitmapProps<T>(getHitmapId: GetHitmapId<T>, children: T[]): ?(HitmapProp[]) {
   if (!getHitmapId || !children || children.length === 0) {
     return undefined;
   }
 
   return children.reduce((memo, marker) => {
-    const hitmapId = getHitmapId(marker);
-    // filter out components that don't have hitmapIds
-    if (hitmapId != null) {
+    if (marker.points) {
```

Comment:
If I'm reading this correctly, this changes the logic in a non-backwards-compatible way: previous if a marker had points (such as a polygon, line strip, or triangle strip) it would receive a single hitmap id...but now we assign each point in those markers it's own hitmap id?  ⚠️